### ISSUE TITLE
0.16.1 crash: missing getProperty() implementation

### ISSUE THREAD

**Issue by akarnokd** [hierynomus/license-gradle-plugin#193]
Hi. I upgraded to 0.16.1 and now my build (Gradle 6.8.3) crashes with the following error:

> Receiver class com.hierynomus.gradle.license.LicenseBasePlugin does not define or inherit an implementation of the resolved method 'abstract java.lang.Object getProperty(java.lang.String)' of interface groovy.lang.GroovyObject.

> Caused by: java.lang.AbstractMethodError: Receiver class com.hierynomus.gradle.license.LicenseBasePlugin does not define or inherit an implementation of the resolved method 'abstract java.lang.Object getProperty(java.lang.String)' of interface groovy.lang.GroovyObject.
>        at com.hierynomus.gradle.license.LicenseBasePlugin$_createExtension_closure2.doCall(LicenseBasePlugin.groovy:73)

0.15.0 still works.

**Comment by NilsOliverLinden** [comment]
It seems Gradle 7.0 and 0.16.1 are compatible.

**Comment by hierynomus** [comment]
Yes, that's what I used when releasing/testing 0.16.1 indeed. I'll check with 6.0+ to see whether something can be done to improve this.

**Comment by rarcher** [comment]
I am also experiencing this issue.

**Comment by rarcher** [comment]
Work-around seems to be to upgrade Gradle and the plugin at the same time, as 0.15.0 seems to be incompatible with Gradle 7.

**Comment by Samuel3** [comment]
@rarcher If you have a bigger project with multiple Gradle plugins it is not always so easy to update the Gradle version due to the plugin constraints. I would really appreciate a fix for Gradle 7.0 as well as the versions below.

**Comment by torgeros** [comment]
something when running version 0.15.0 with latest gradle:

```
Executing tasks: [downloadLicenses] in project XXXXXXXXXXXXXXXXXXXXXXXXX

> Task :app:downloadLicenses

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 141ms
1 actionable task: 1 executed
14:32:12: Task execution finished 'downloadLicenses'.
```

**Comment by xtreme-shane-lattanzio** [comment]
I am also running into this.Are there plans to have the new version be backwards compatible with older versions of gradle?
